docs/html/conf.py

```diff
@@ -12,22 +12,30 @@
 sys.path.insert(0, docs_dir)
 
 # -- General configuration ------------------------------------------------------------
-
 extensions = [
-    # first-party extensions
-    "sphinx.ext.autodoc",
-    "sphinx.ext.todo",
-    "sphinx.ext.extlinks",
-    "sphinx.ext.intersphinx",
```

Comment:
These are all built-in so there's no value moving to conditional (and it makes things harder to reason about)

Reply:
I think there might be value in skipping some code paths, though.